test.sh

```diff
@@ -0,0 +1,15 @@
+result=$(curl -k -X POST -H 'content-type: application/json' https://localhost:4000/api/v1/scenes -d '
+{"scene": {
+	"name": "test",
+	"description": "a test scene",
+	"attribution_name": "a guy",
+	"attribution_link": "twitter.com/a_guy",
+	"author_account_id": "123123",
+	"upload_id": "234234"
+}}')
+echo "$result"
+
+scene_id=$(echo $result | jq -r '.scene_id')
+echo "$scene_id"
+
+curl -k "https://localhost:4000/api/v1/scenes/$scene_id"
```

Comment:
I'll remove this for the final merge, but should I add some actual tests instead?

Reply:
i think an integration test would be a good idea -- some on me for not getting the other ones rigged up